src/MultiversionManager.php

```diff
@@ -495,7 +531,8 @@ protected function prepareContentForMigration($entity_types, MultiversionMigrati
         if ($storage->getEntityTypeId() === 'file') {
           $migration->copyFilesToMigrateDirectory($storage);
         }
-        $migration->migrateContentToTemp($storage->getEntityType());
+        $process = $migration->getFieldMap($entity_type, $op, self::TO_TMP);
```

Comment:
The parameter name is ambiguous because the method returns the filed map, not a process.